news/_posts/2019-09-10-the-review-process.md

```diff
@@ -0,0 +1,89 @@
+
+If something is wrong, Prestonbot will write a comment in the pull request to tell you what to fix.
+
+### Code review
+
+Now that all the bots have done their job, humans finally come on the stage ! Your Pull Request will be reviewed by a member of the Core Team.
+
+#### What do we check in a code review?
+
+When a Core Team member opens a pull request, he will review it and decide whether it should be accepted, or it needs changes, or it cannot be accepted.
+
+The review process is quite complete in order to make sure that PrestaShop codebase gets better with each contribution. Here is all the things we look for in a Pull Request, when reviewing it:
+
+- Of course we start by checking that the code is correct. This means both from a behavior point of view than from a technical point of view. This is simply an assessment of the quality of the Pull Request code, just like it happens in a lot of software teams. We check the code is correct, we check the code uses the right functions, we check the code handles expectable edge-cases, we check the code has no vulnerability, we check the code scales well ... We also keep in mind that PrestaShop is a CMS and consequently must be extensible to allow developers to easily customize the behavior, so we make sure to provide relevant extension points.
+- We also assess the readability of the code. There is a statement that says 'when a code file is opened by a developer, 9 times out of 10 it will only to be read, not to be modified'. Because PrestaShop is a huge and complex codebase and because it has so many people reading through it, it is very important that we make the code as readable as possible. This is obtained by adding comments, choosing carefully function and variable names and building an architecture that makes sense so it is easy to grasp and navigate for people who have never worked on it before.
+- We also check that best practices are implemented into the Pull Request, be it standard conventions or practices like [PSR](https://www.php-fig.org/psr/) or best security recommandations like the [OWASP](https://www.owasp.org/) recommandations. When people use PrestaShop to build a custom website, it is likely that they will follow the practices they see implemented in the Core so we try to think of the code we merge as an example that people will use.
+- PrestaShop has grown huge over the years, both as a codebase and as a software. There are hundred of features built in the software, but it is not often that people use them all. That is why some contributions sometimes need to be improved because they are not compatible with one of the existing features of the software. Common examples are the multi-store mode or the RTL (Right-To-Left) mode, two feature that a lot of developers do not use (because it adresses specific merchant or countries needs) and consequently when they submit a Pull Request, they do not check whether the modification they submit is compatible with these modes.
```

Comment:
```suggestion
- PrestaShop has grown huge over the years, both as a codebase and as a software. There are hundreds of features built in the software, and some are more commonly used than others. Some contributions sometimes need to be reworked because they did not take into account one of the less popular features of the software, or are not compatible with them. Common examples are the multi-store mode or the RTL (Right-To-Left) mode, two features that adresses very specific needs and that many developers are not aware of.
```